### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  com.microsoft.azure.toolkit.lib.common.operation.AzureOperationException: prepare before run configuration

### ISSUE THREAD

**Issue by cforce** [microsoft/azure-tools-for-java#6382]
IntelliJ build version: 2021.3.2 IU-213.6777.52
OS: Windows 11
JDK: JetBrains s.r.o. 11.0.13
Plugin version: 3.61.1-2021.3
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,7,7,2561x1393,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=com.microsoft.azure.toolkit.lib.common.operation.AzureOperationException: prepare before run configuration
	at com.microsoft.azure.toolkit.lib.common.operation.AzureOperationAspect.afterThrowing(AzureOperationAspect.java:77)
	at com.microsoft.azure.toolkit.intellij.connector.Connection.prepareBeforeRun(Connection.java:96)
	at com.microsoft.azure.toolkit.intellij.connector.ConnectionRunnerForRunConfiguration.lambda$executeTask$0(ConnectionRunnerForRunConfiguration.java:87)
	at com.microsoft.azure.toolkit.intellij.connector.ConnectionRunnerForRunConfiguration.executeTask(ConnectionRunnerForRunConfiguration.java:87)
	at com.microsoft.azure.toolkit.intellij.connector.ConnectionRunnerForRunConfiguration.executeTask(ConnectionRunnerForRunConfiguration.java:43)
Caused by: com.microsoft.azure.toolkit.lib.auth.exception.AzureToolkitAuthenticationException: you are not signed-in.

**Comment by wangmingliang-ms** [comment]
thanks,
As the message said, **you are not signed in**, which causes the toolkit to fail getting the details of the connected Azure resources.

Anyway, we'll catch the exception and show it properly as an error Notification to tell the cause explicitly.
this fix would be shipped in the coming v3.62.0

**Comment by wangmingliang-ms** [comment]
the fix has been released in the latest v3.62.0, please try and help confirm, thanks.
